src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUY_OPERATION = "buy";
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        List<String> readedFile;
```

Comment:
The variable name 'readedFile' is a grammatical error, and it is also not descriptive enough. Consider renaming it to 'linesFromFile' to better reflect its purpose.